src/mistral_common/tokens/tokenizers/base.py

```diff
@@ -311,11 +344,14 @@ def encode_instruct(self, request: InstructRequestType) -> TokenizedType:
         """
 
     @abstractmethod
-    def decode(self, tokens: List[int]) -> str:
+    def decode(self, tokens: List[int], special_token_policy: Optional[SpecialTokenPolicy] = None) -> str:
         r"""Convert token ids to string
 
         Args:
             tokens: The token ids to decode.
+            special_token_policy: The policy to use for special tokens.
+                Passing `None` is deprecated and will be changed
```

Comment:
Same change as in https://github.com/mistralai/mistral-common/pull/99/files#r2144815908